src/components/Popover/Popover.tsx

```diff
@@ -1,12 +1,13 @@
 import { Popover as HeadlessPopover } from '@headlessui/react';
 import type { Options as PopperJSOptions } from '@popperjs/core';
+import clsx from 'clsx';
 import * as React from 'react';
```

Comment:
nit, while you're in here:
```suggestion
import * as React from 'react';
```
just making it look like the other components lol